kuttl-test-helm-certmanager.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+artifactsDir: ./tests/_build/artifacts/
+testDirs:
+  - ./tests/sidecar/
+  - ./tests/operator/
+timeout: 150
+parallel: 4
+startKIND: true
+kindNodeCache: true
+kindContainers:
+  - registry.localhost:5000/sumologic/tailing-sidecar-operator:test
+  - registry.localhost:5000/sumologic/tailing-sidecar:test
+commands: 
+  - command: make -C ./operator deploy-cert-manager
+  - script: helm upgrade --install test-release ./helm/tailing-sidecar-operator -f ./helm/tests/values.withCertManager.yaml -n tailing-sidecar-system --create-namespace
```

Comment:
what is difference between command and script?

Reply:
Script uses a full shell, so you can do pipes for example. I don't think it's actually needed here, this command used to do some clever env var expansion. I can switch it to `command` now.